README.md

```diff
@@ -548,8 +548,8 @@ terragrunt = {
 ```
 
 The `include` block tells Terragrunt to use the exact same Terragrunt configuration from the `terraform.tfvars` file
-specified via the `path` parameter. It behaves exactly as if you had copy/pasted the Terraform configuration from 
-the included file's terragrunt = { ... } configuration into `mysql/terraform.tfvars`, but this approach is much easier to maintain! Including variables defined outside of the terragrunt block is discussed in #132.
+specified via the `path` parameter. It behaves exactly as if you had copy/pasted the Terraform configuration from
+the included file's terragrunt = { ... } configuration into `mysql/terraform.tfvars`, but this approach is much easier to maintain! Including variables defined outside of the terragrunt block is discussed in [#132](https://github.com/gruntwork-io/terragrunt/issues/132).
```

Comment:
Unlike in issues or comments, in MD files #issues are not automatically linked. Therefore added explicit link to the issue.